### ISSUE TITLE
[BUG] navbar task counter doesn't update upon adding a new task

### ISSUE THREAD

**Issue by dylantrann** [cse110-sp24-group14/cse110-sp24-group14#127]
There is a bug in which the "x more to go!" subtext on the navbar is not updated after a new task is created. 

This bug was found by Trey in correlation to #101. We plan to fix this bug after #101 gets merged and more task functionality is accessible on a newer branch.
